packages/core/src/styles/layout/_alignments.scss

```diff
@@ -19,13 +19,11 @@
 }
 
 @mixin generateHorizontalVerticalItemStretchShrink($breakpoint: null) {
-  [#{$layout}*='#{$align}#{$breakpoint}:#{$stretch}'],
-  ::slotted([#{$layout}*='#{$align}#{$breakpoint}:#{$stretch}']) {
+  [#{$layout}*='#{$align}#{$breakpoint}:#{$stretch}'] {
     flex-grow: 1 !important;
   }
 
-  [#{$layout}*='#{$align}#{$breakpoint}:#{$shrink}'],
-  ::slotted([#{$layout}*='#{$align}#{$breakpoint}:#{$shrink}']) {
+  [#{$layout}*='#{$align}#{$breakpoint}:#{$shrink}'] {
```

Comment:
The slotted styles were not being used anywhere and I don't believe they worked properly either.